apps/store/extensions/app/social-reviews/modules/review-api.js

```diff
@@ -29,6 +29,7 @@ var api = {};
     var HTTP_GET_METHOD = 'GET';
     var HTTP_POST_METHOD = 'POST';
     var HTTP_DELETE_METHOD = 'DELETE';
+    var HTTP_PUT_METHOD = 'PUT';
```

Comment:
Please consider making constant variables for Content-Type and application/json. Because those strings have more occurrences than GET and POST.